tests/unit/model/test_model_item.py

```diff
@@ -18,20 +18,50 @@
 
 import pytest
 
+from structurizr.model import Model, SoftwareSystem, SoftwareSystemIO
 from structurizr.model.model_item import ModelItem
 
 
+@pytest.fixture(scope="function")
+def model() -> Model:
+    return Model()
+
+
 class ConcreteModelItem(ModelItem):
     """Implement a concrete `ModelItem` class for testing purposes."""
 
     pass
 
 
 @pytest.mark.parametrize(
-    "attributes", [{}],
+    "attributes",
+    [{}],
 )
 def test_model_item_init(attributes):
     """Expect proper initialization from arguments."""
     model_item = ConcreteModelItem(**attributes)
     for attr, expected in attributes.items():
         assert getattr(model_item, attr) == expected
+
+
+def test_get_tags_when_there_are_no_tags(model: Model):
```

Comment:
Please add a docstring to the test case. Below a suggestion for changing the name.

```suggestion
def test_default_element_tags_order(model: Model):
```

Reply:
Was being consistent with the Java test name.  Happy either way.

Reply:
Done.

Reply:
I didn't realize that, sorry. That might be a good reason to keep the name. Just so we know that there is a consistent set of tests from Java that the Python client passes?

Reply:
I prefer the more meaningful name you suggested, so have compromised and referenced the Java test name in the docstring so we can still trace it.